testing/dart/channel_buffers_test.dart

```diff
@@ -192,13 +192,13 @@ void main() {
   test('handle garbage', () async {
     final ui.ChannelBuffers buffers = ui.ChannelBuffers();
     expect(() => buffers.handleMessage(_makeByteData('asdfasdf')),
-           throwsException);
+           throwsA(isInstanceOf<Exception>()));
```

Comment:
It's also trivial to define `throwsException`, `throwsAssertionError`, which are things that will probably make it less confusing for people used to writing Dart tests on the framework side.

Reply:
Done.